### PR TITLE
Adds spray secondary flamer for tank

### PR DESCRIPTION
**Pull request by TiviPlus**
## About The Pull Request

this is basically the tye style flamethrower that shoots fast but does 1 tile as in the updated doc

## Why It's Good For The Game

Second secondary weapon option lol

## Changelog
:cl: TiviPlus, Kuro, Tyeagg
add: Adds spray secondary flamer for tank
/:cl:

### CODE REVIEW

**Review by Kuro020 — APPROVED** [review]
Where's the video noob

File: code/modules/projectiles/ammo_types/miscellaneous_ammo.dm [hunk 1]
```diff
@@ -91,7 +91,7 @@
 	name = "flame"
 	icon_state = "pulse0"
 	hud_state = "flame"
-	hud_state_empty = "flame_empty"
+	hud_state_empty = "§"
```
Comment (TiviPlus) [inline code/modules/projectiles/ammo_types/miscellaneous_ammo.dm:94]:
```suggestion
	hud_state_empty = "flame_empty"
```

**Comment by Wisemonster** [comment]
Should probably rename it to the extended flamer or stream flamer, considering spray implies it would act like the spray nozzle, but it sounds like it acts more like a continuous extended nozzle.

**Comment by TiviPlus** [comment]
its already mostly names secondary flamer in tank so we're good

**Review by Lumipharon — APPROVED** [review]

**Pull request merged**